README.md

```diff
@@ -11,7 +11,7 @@ threading fashion or asyncio/trio/curio.
 ## Usage
 
 For a reference implementation see
-[wumpy-gateway](https://github.com/Bluenix2/wumpy/library/wumpy-gateway/wumpy/gateway/shard.py).
+[wumpy-gateway](https://github.com/Bluenix2/wumpy/blob/main/library/wumpy/gateway/shard.py).
```

Comment:
I don't think I read the link closely enough, it seems like it should be:
https://github.com/Bluenix2/wumpy/blob/main/library/wumpy-gateway/wumpy/gateway/shard.py

Reply:
Changed 👍